### DISCUSSION TITLE
What is the difference between native-maven-plugin and native-image-maven-plugin plugin ?

### DISCUSSION THREAD

**Discussion by xzxiaoshan** [oracle/graal#4765] [Q&A] [answered]
What is the difference between native-maven-plugin and native-image-maven-plugin plugin ?

3Q~

---beginner personal

**Comment by alina-yur** [comment, marked as answer]
Hi @xzxiaoshan, `native-image-maven-plugin plugin` is no longer updated — it was replaced by a new `native-maven-plugin` from the [Native Build Tools](https://graalvm.github.io/native-build-tools/latest/index.html) project. Please use this one, see the [docs](https://graalvm.github.io/native-build-tools/latest/maven-plugin.html).
